ui/apps/platform/package.json

```diff
@@ -124,6 +124,7 @@
         "@testing-library/jest-dom": "^5.15.0",
         "@testing-library/react": "^12.1.2",
         "@testing-library/react-hooks": "^7.0.2",
+        "@testing-library/user-event": "^14.2.1",
```

Comment:
Yes, please update any shared dependencies in package.json files:
* apps/platform
* packages/tailwind-config
* packages/ui-components

Keep it simple for yarn because we have 2-level tree monorepo instead of 1-level flat monorepo.

Reply:
Thinking about your question more carefully, **ordinary** dependencies need to be consistent in all package.json files, not just in yarn.lock file, because infra project sees package.json of the 2 reusable packages only, therefore cannot inherit a newer version from apps/platform package.json file, as has happened here for the dev dependency. For example, React 18 upgrade. Having said that, updates to infra have slowed down in 2022.